polling_stations/apps/dashboard/templates/dashboard/council_detail.html

```diff
@@ -0,0 +1,82 @@
+{% extends "base.html" %}
+{% load humanize %}
+{% load i18n %}
+
+{% block page_title %}{{ council }} - {% trans "Dashboard" %}{% endblock page_title %}
+
+{% block content %}
+<h1>{{ council }}</h1>
+
+    <section>
```

Comment:
We don't need to get really bogged down in styling for an internal tool, but if you make all these `<sections>` `<section class="card">` that will make a huge difference to appearance.

Reply:
:+1:

Reply:
Done